x/subspaces/client/cli/tx.go

```diff
@@ -476,21 +476,17 @@ Multiple permissions must be specified separating them with a comma (,).`, FlagD
 				return err
 			}
 
-			permissions, err := cmd.Flags().GetStringSlice(FlagPermissions)
+			flagPermissions, err := cmd.Flags().GetStringSlice(FlagPermissions)
```

Comment:
The example of the command should be updated to `%s tx subspaces groups create 1 "Admins" \
  --description "Group of the subspace admins" \
  --permissions "WRITE,MODERATE_CONTENT,SET_USER_PERMISSIONS" \
  --from alice`.